pyuvdata/tests/test_uvbeam.py

```diff
@@ -411,6 +327,50 @@ def test_az_za_to_healpix():
     power_beam.pixel_coordinate_system = 'sin_zenith'
     nt.assert_raises(ValueError, power_beam.az_za_to_healpix)
 
+    # Now check Efield interpolation
+    efield_beam = UVBeam()
+    efield_beam.read_cst_beam(cst_files[0], beam_type='efield', frequency=150e6,
+                              telescope_name='TEST', feed_name='bob',
+                              feed_version='0.1', feed_pol=['x'],
+                              model_name='E-field pattern - Rigging height 4.9m',
+                              model_version='1.0')
+    interp_then_sq = efield_beam.az_za_to_healpix(inplace=False)
+    interp_then_sq.efield_to_power(calc_cross_pols=False)
+
+    # convert to power and then interpolate to compare.
+    # Don't use power read from file because it has rounding errors that will dominate this comparison
+    sq_then_interp = efield_beam.efield_to_power(calc_cross_pols=False, inplace=False)
+    sq_then_interp.az_za_to_healpix()
+
+    # square then interpolate is different from interpolate then square at a
+    # higher level than normally allowed in the equality.
+    # We can live with it for now, may need to improve it later
+    diff = np.abs(interp_then_sq.data_array - sq_then_interp.data_array)
+    nt.assert_true(np.max(diff) < 0.5)
+    reldiff = diff / sq_then_interp.data_array
+    nt.assert_true(np.max(reldiff) < 0.05)
+
+    tols = [0.05, 0]
+    nt.assert_true(np.allclose(sq_then_interp.data_array, interp_then_sq.data_array,
+                               rtol=tols[0], atol=tols[1]))
```

Comment:
This shows up in a few places... is this assert redundant with the two before it (lines 349 and 351)?

Reply:
yes, it's redundant, I'll remove it.